docs/migration/3_0.md

```diff
@@ -217,3 +227,13 @@ Now it is:
 - `ShippingAddressComponent` no longer uses `CheckoutConfigService` and `RoutingService`. These services usage was replaced with the corresponding methods form `CheckoutStepService`. This service needs to be provided to `ShippingAddressComponent`.
 - `MultiCartService` now requires the additional provider `UserIdService`.
 - `PageSlotComponent` no longer uses `CmsComponentsService`. This service usage was replaces with the `PageSlotService`.
+- `ForbiddenHandler` now uses `GlobalMessageService`, `AuthService`, `OccEndpointsService`.
+- `CheckoutPaymentService` no longer uses `AuthService`. This service usage was replaced with the corresponding methods from `UserIdService`.
+- `CheckoutService` no longer uses `AuthService`. This service usage was replaced with the corresponding methods from `UserIdService`.
+- `CustomerCouponService` no longer uses `AuthService`. This service usage was replaced with the corresponding methods from `UserIdService`.
+- `OrderReturnRequestService` no longer uses `AuthService`. This service usage was replaced with the corresponding methods from `UserIdService`.
+- `UserAddressService` no longer uses `AuthService`. This service usage was replaced with the corresponding methods from `UserIdService`.
+- `UserConsentService` no longer uses `AuthService`. This service usage was replaced with the corresponding methods from `UserIdService`.
```

Comment:
It still requires `AuthService`. Only inform that UserIdService is also required there.